framework/src/utils/FormattedTable.C

```diff
@@ -112,7 +112,8 @@ void
 FormattedTable::addData(const std::string & name, Real value, Real time)
 {
   _data[time][name] = value;
-  _column_names.insert(name);
+  if (std::find(_column_names.begin(), _column_names.end(), name) == _column_names.end())
```

Comment:
This is the only bit I don't like about this PR, but I am not sure of a better solution or if it is all that important with respect to performance.